src/components/UI/molecules/Footer/About/About.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Title from '<atoms>/Title/Title';
+import Text from '<atoms>/Text/Text';
+
+const About = ({ title, texts }) => (
+  <FlexContainer
+    backgroundColor="primary"
+    padding="zero"
+  >
+    <Fragment>
```

Comment:
Why do you have a Fragment here when everything is nested in the `TextContainer`